server/src/server.ts

```diff
@@ -93,31 +95,26 @@ connection.onInitialized(() => {
 })
 
 documents.onDidOpen(event => {
-  connection.console.log(
-    `Language server received an open event for ${event.document.uri}`
-  )
-})
-
-documents.onDidClose(event => {
-  connection.console.log(
-    `Language server received a close event for ${event.document.uri}`
-  )
-})
-
-documents.onDidChangeContent(async event => {
```

Comment:
Yeah, the bundle model only updates with saved content. Shouldn't be a huge difference in user experience since gitpod autosaves pretty frequently anyways.